src/modules/netinstall/netinstall.conf

```diff
@@ -1,7 +1,9 @@
 ---
 # This is the URL that is retrieved to get the netinstall groups-and-packages
 # data (which should be in the format described in netinstall.yaml).
-groupsUrl: http://chakraos.org/netinstall.php
+# groupsUrl: http://example.org/netinstall.php
+# or it can be a locally installed file (add netinstall.yaml to ./CMakeLists.txt)
```

Comment:
Don't mention changing CMakeLists.txt, that's really not the way downstream should be putting configuration files into the live system.